src/main/scala-2.13-/scala/concurrent/stm/compat/package.scala

```diff
@@ -0,0 +1,31 @@
+package scala.concurrent.stm
```

Comment:
I would rather add a dependency to scala-collection-compat instead of re-defining its content here.

Reply:
Let's ask the maintainer what is the dependency policy.

Reply:
@nbronson @Sciss What do you think of depending on [scala-collection-compat](https://github.com/scala/scala-collection-compat/)?